sample chaincodes in java failed installing for fabric 2.3.2

with fabric 2.3.2, it raised following error. I didn't check with other fabric versions...

```
minifab -i 2.3.2 -e true -l java

Minifab Execution Context:
    FABRIC_RELEASE=2.3.2
    CHANNEL_NAME=mychannel
    PEER_DATABASE_TYPE=golevel
    CHAINCODE_LANGUAGE=java
    CHAINCODE_NAME=simple
    CHAINCODE_VERSION=1.0
    CHAINCODE_INIT_REQUIRED=true
    CHAINCODE_PARAMETERS="init","a","200","b","300"
    CHAINCODE_PRIVATE=false
    CHAINCODE_POLICY=
    TRANSIENT_DATA=
    BLOCK_NUMBER=newest
    EXPOSE_ENDPOINTS=true
    CURRENT_ORG=org0.example.com
    HOST_ADDRESSES=xxx.xxx.xxx.xxx
    WORKING_DIRECTORY: /mnt/minifabric

# Running operation: ******************************************
  cc install
.......
# Run the chaincode install script on cli container ***********
  non-zero return code
  Error: chaincode install failed with status: 500 - failed to invoke backing implementation of 'InstallChaincode': could not build chaincode: docker build failed: docker image build failed: docker build failed: Error returned from build: 1 "+ INPUT_DIR=/chaincode/input
  + OUTPUT_DIR=/chaincode/output
  ++ mktemp -d
  + TMP_DIR=/tmp/tmp.DINgPo
  ++ find /chaincode/input -name .jar
  ++ paste -s -d : -
  + JARS=
  ++ find /chaincode/input -name '*.jar'
  ++ wc -l
  + NUM_JARS=0
  + for DIR in ${INPUT_DIR} ${INPUT_DIR}/src
  + '[' -f /chaincode/input/build.gradle -o -f /chaincode/input/build.gradle.kts ']'
  + '[' -f /chaincode/input/pom.xml ']'
  + for DIR in ${INPUT_DIR} ${INPUT_DIR}/src
  + '[' -f /chaincode/input/src/build.gradle -o -f /chaincode/input/src/build.gradle.kts ']'
  + buildGradle /chaincode/input/src /chaincode/output
  + echo 'Copying from /chaincode/input/src to /tmp/tmp.DINgPo'
  Copying from /chaincode/input/src to /tmp/tmp.DINgPo
  + cd /chaincode/input/src
  + tar cf - .
  + cd /tmp/tmp.DINgPo
  + tar xf -
  + cd /tmp/tmp.DINgPo
  + echo 'Gradle build'
  + '[' -f ./gradlew ']'
  Gradle build
  + /root/chaincode-java/gradlew build shadowJar -x test
  Downloading https://services.gradle.org/distributions/gradle-7.0-bin.zip
  ..........10%...........20%...........30%..........40%...........50%...........60%...........70%..........80%...........90%...........100%
  
  Welcome to Gradle 7.0!
  
  Here are the highlights of this release:
   - File system watching enabled by default
   - Support for running with and building Java 16 projects
   - Native support for Apple Silicon processors
   - Dependency catalog feature preview
  
  For more details see https://docs.gradle.org/7.0/release-notes.html
  
  Starting a Gradle Daemon (subsequent builds will be faster)
  
  FAILURE: Build failed with an exception.
  
  * Where:
  Build file '/tmp/tmp.DINgPo/build.gradle' line: 28
  
  * What went wrong:
  A problem occurred evaluating root project 'fabric-chaincode-example-gradle'.
  > Could not find method compile() for arguments [{group=org.hyperledger.fabric-chaincode-java, name=fabric-chaincode-shim, version=1.+}] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.
  
  * Try:
  Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
  
  * Get more help at https://help.gradle.org
  
  BUILD FAILED in 1m 25s
  "

# STATS *******************************************************
minifab: ok=307	failed=1	
```

when I tested with following patched build.gradle , it raised another error in cc initalize phase (dependency jars not found in fabric-javaenv log). it may be better to bundle gradlew (gradle wrapper) to freeze gradle version.
```patch
dependencies {
-    compile group: 'org.hyperledger.fabric-chaincode-java', name: 'fabric-chaincode-shim', version: '1.+'
+    compileOnly group: 'org.hyperledger.fabric-chaincode-java', name: 'fabric-chaincode-shim', version: '1.+'
-    testCompile group: 'junit', name: 'junit', version: '4.12'
}
```

--- a/chaincode/simple/java/build.gradle
+++ b/chaincode/simple/java/build.gradle
@@ -25,8 +25,10 @@ repositories {
 }
 
 dependencies {
-    compile group: 'org.hyperledger.fabric-chaincode-java', name: 'fabric-chaincode-shim', version: '1.+'
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    compileOnly group: 'org.hyperledger.fabric-chaincode-java', name: 'fabric-chaincode-shim', version: '1.+'
+    implementation group: 'org.hyperledger.fabric-chaincode-java', name: 'fabric-chaincode-shim', version: '1.+'
+    testImplementation group: 'junit', name: 'junit', version: '4.12'
+
 }
 
 shadowJar {

